src/engines/v8/v8.zig

```diff
@@ -197,13 +197,9 @@ pub const Env = struct {
             // TODO: this is an horrible hack
             if (comptime T_refl.isException()) {
                 const script = T_refl.name ++ ".prototype.__proto__ = Error.prototype";
-                const res = try self.execTryCatch(
-                    alloc,
-                    script,
-                    "errorSubclass",
-                );
-                defer res.deinit(alloc);
-                if (!res.success) return error.errorSubClass;
+                _ = self.exec(script, "errorSubclass") catch {
```

Comment:
This override of error was done before, here I just change the call function because `execTryCatch` was removed.

I don't want to change this as it may have implications who are not part of the scope of this PR. I can add a comment though.